.github/workflows/coverage.yaml

```
@@ -3,60 +3,94 @@ name: Evmone Coverage Report
 on:
   pull_request:
     paths:
-      - "converted-ethereum-tests.txt" # This triggers the workflow only for changes in file.txt
+      - 'tests/**' # This triggers the workflow for any changes in the tests folder
+      - '!tests/prague/**'  # This excludes changes in 'tests/prague'
```

Let's add osaka up front as it's in main:
```suggestion
      - '!tests/prague/**'  # exclude changes in 'tests/prague'
      - '!tests/osaka/**'  # excludes changes in 'tests/osaka'
```